macros/migrate_v1_to_v2.sql

```diff
@@ -0,0 +1,117 @@
+{% macro migrate_artifacts_v1_to_v2() %}
+
+{% set src_dbt_artifacts = source('dbt_artifacts', 'artifacts') %}
+{% set src_results = source('dbt_artifacts', 'dbt_run_results') %}
+{% set src_results_nodes = source('dbt_artifacts', 'dbt_run_results_nodes') %}
+{% set src_manifest_nodes = source('dbt_artifacts', 'dbt_run_manifest_nodes') %}

```

Comment:
```suggestion
{% set src_manifest_nodes = source('dbt_artifacts', 'dbt_manifest_nodes') %}

```

Just noticed this, may need updating elsewhere. Nitpick but just avoids confusion, the artifacts are named 'run_results' and 'manifest'

Reply:
Good call - it does impact a few things to change - but much easier to change at this stage than later down the line.